Index: web/modules/contrib/charts/modules/charts_api_example/templates/charts_api_example.html.twig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/modules/contrib/charts/modules/charts_api_example/templates/charts_api_example.html.twig b/modules/contrib/charts/modules/charts_api_example/templates/charts_api_example.html.twig
--- a/modules/contrib/charts/modules/charts_api_example/templates/charts_api_example.html.twig
+++ b/modules/contrib/charts/modules/charts_api_example/templates/charts_api_example.html.twig	(date 1648994761492)
@@ -1,4 +1,3 @@
 {% set library, height, width, height_units, width_units = 'charts_' ~ library ~ '/' ~ library, options.height, options.width, options.height_units, options.width_units %}
 {{ attach_library("#{ library }") }}
-<div {{ attributes }} {{ content_attributes }}
-        style="{% if width is not empty %}width:{{ width }}{{ width_units }};{% endif %}{% if height is not empty %}height:{{ height }}{{ height_units }};{% endif %}"></div>
+<div {{ attributes }} {{ content_attributes }} style="width: 850px; height: 500px;"></div>
